las_geoh5/import_files/driver.py

```diff
@@ -95,14 +89,15 @@ def run(filepath: str):  # pylint: disable=too-many-locals
         ifile.data["name"],
     )
     begin_saving = time()
+
+    name_options = NameOptions(**{k: v for k, v in ifile.data.items() if v is not None})

```

Comment:
Another "pydantic" way to do this would be to add a  @model_validator on your class, and deal with it there. See suggestion below